modules/nixos/nix-build-distributor.nix

```
@@ -25,6 +27,26 @@
       speedFactor = 10;
     }
 
+    # macos-03
+    {
+      hostName = "142.132.140.224";
+      sshUser = "builder";
+      protocol = "ssh-ng";
+      system = "aarch64-darwin";
+      maxJobs = 8;
+      supportedFeatures = config.nix.settings.experimental-features;
+    }
+
+    # macos-04
+    {
+      hostName = "167.235.38.111";
+      sshUser = "builder";
+      protocol = "ssh-ng";
+      system = "aarch64-darwin";
```

```suggestion
      system = "x86_64-darwin";
```

i'd like to balance capacity and improve resilience for `x86_64-darwin`